specification/columns/tags.md

```diff
@@ -1,6 +1,6 @@
 # Tags
 
-The Tags column represents the set of tags assigned to [*tag sources*](#glossary:tag-source) that also account for potential provider-defined or user-defined tag evaluations.  Tags are commonly used for scenarios like adding business context to billing data to identify and accurately allocate charges.
+The Tags column represents the set of tags assigned to [*tag sources*](#glossary:tag-source) that also account for potential provider-defined or user-defined tag evaluations.  In FOCUS, "Tag" is a generic term to describe metadata that can be attached to a resource, account, etc. by a user or provider to help identify it. Tags may also be referred to by providers by other terms, such as labels or networks tags. Tags are commonly used for scenarios like adding business context to billing data to identify and accurately allocate charges. 
```

Comment:
Couple suggestions:
1. I'd propose moving sentence, "In FOCUS, "Tag" is ..." to adjust the "Tag" term in the [Glossary](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/blob/working_draft/specification/glossary.md?plain=1#L127-L129) and adding a link to this term in the very first sentence of the description.
2.  I like this sentence, "Tags may also be referred to ..." but after "Tags are commonly used ..."

Reply:
Great suggestions, thanks Chris! Tags was already in the glossary so I just removed it here. Also repositioned the sentence.